src/test/end-to-end/codecept.conf.js

```diff
@@ -25,8 +25,30 @@ exports.config = {
       minTimeout: 2000
     }
   },
-  include: { I: './steps_file.js' },
+  include: { 
+    I: './steps_file.js', 
+    CitizenLoginPage: './pages/C100-Rebuild/CitizenLoginPage.js',
+    CreateApplication: './pages/C100-Rebuild/CreateApplication.js',
+    CaseNameAndPostCode: './pages/C100-Rebuild/CaseNameAndPostCode.js',
+    ScreeningQuestions: './pages/C100-Rebuild/ScreeningQuestions.js',
+    GoToMiam: './pages/C100-Rebuild/GoToMiam.js',
+    TypeOfOrder: './pages/C100-Rebuild/TypeOfOrder.js',
+    UrgencyWithoutNotice: './pages/C100-Rebuild/UrgencyWithoutNotice.js',
+    ChildrenDetails: './pages/C100-Rebuild/ChildrenDetails.js',
+    ApplicantDetails: './pages/C100-Rebuild/ApplicantDetails.js',
+    RespondentDetails: './pages/C100-Rebuild/RespondentDetails.js',
+    OtherPersonDetails: './pages/C100-Rebuild/OtherPersonDetails.js',
+    OtherProceedings: './pages/C100-Rebuild/OtherProceedings.js',
+    SafetyConcerns: './pages/C100-Rebuild/SafetyConcerns.js',
+    InternationElements: './pages/C100-Rebuild/InternationElement.js',
+    ReasonableAdjustments: './pages/C100-Rebuild/ReasonableAdjustments.js',
+    HelpWithFees: './pages/C100-Rebuild/HelpWithFees.js',
+    CheckYourAnswers: './pages/C100-Rebuild/CheckYourAnswers.js',
+    CheckYourAnswersSimple: './pages/C100-Rebuild/CheckYourAnswersSimple',
+    ConsentOrder: './pages/C100-Rebuild/ConsentOrder.js'
```

Comment:
These are included in the steps_file, so probably not needed here

Reply:
I've updated the codecept.conf.js to remove these